_posts/2025-04-28-towards-more-rigorous-llm-evals.md

```diff
@@ -199,8 +201,14 @@ $$N \cdot p_m \cdot (1-p_m).$$
 
 This variance is maximised when $p_m=1/2$ and goes to $0$ as $p_m$ approaches $0$ or $1$.
 
+<!-- Probably use expected variation to follow the subsection title and also to avoid confusion with Normal distribution. -->
 To quantify what constitutes "normal" variation under our assumption, we can construct confidence intervals (CI) for the point estimates of $p_m$ (these are reported in the second column of Table 1 in the Appendix of the paper). 
+<!-- Perhpas better to phrase it as the score on the GSM8K 100 questions, reported in Table 1, can be used as point estimate of $p_m$ on the GSM8K dataset. To make it clear that the authors do not report it as a point estimate of $p_m$. Af first reading it gives an impression that the paper also talks about Binomial distribution and so it reports the success probability in some table. Also we need to be clear about for which dataset with estimate the $p_m$, GSM8K or GSM-Symbolic -->
 There are different ways to construct CI for the [Binomial proportion](https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval). Figure XX below shows [Wilson score](https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Wilson_score_interval) intervals. (See Appendix for more results.)
+<!-- Needs more text explaining exactly what the figure shows, and also we need that text in the figure's caption. For example, what is the CI around the GSM-Symbolic. Also not sure if it is good to mix the two like this. The reds are estimated CI and the blues are, I assume, just one standard deviation from the mean, as reported in Table 1 of the paper. It is a bit complicated to decide what is best here. Some of the options are: 
+  1) report 1.96 times the standard deviation, to get 95% CI (perhaps this is already what the figures shows) but this assumes Normal distribution.
+  2) ignore the standard deviation from the Table 1 of the paper, and just treat the reported means as a new set of $p_m^{\prime}$ and then do the CI the same you did them for the reds, using the Wilson score. 
+Whatever you decide is best, make it clear in the text and in the Figure's caption. -->
```

Comment:
yeah agree it needs some explanation.

Actually they are both confidence intervals; one is on the point estimate on GSM8K (N=100 questions), the other is on Symbolic, pooling the 50 datasets of 100 questions into one (so N=5000 questions, hence the smaller width).

I chose to show both of them to be able to roughly perform the two-sided hypothesis test from that chart

Reply:
I think you're right though, it can be confusing, will think about it

Reply:
> yeah agree it needs some explanation.
> 
> Actually they are both confidence intervals; one is on the point estimate on GSM8K (N=100 questions), the other is on Symbolic, pooling the 50 datasets of 100 questions into one (so N=5000 questions, hence the smaller width).
> 
> I chose to show both of them to be able to roughly perform the two-sided hypothesis test from that chart

I think you cannot pool the 50 datasets like that. We have 50 Binomial samples with N=100, not one Binomial sample with N=5000. I think is better if you just use the average over the 50 as an estimate of p and set N=100. The width of the CI for the two distributions (GSM8K and Symbolic) should be similar. I realise it might be too late for this paper though.

Reply:
On second thought, I think it is best if that chart is as it was in the original version of the blog post, where it showed just the average of the Symbolic as a single point. It is a clearer figure and it conveys what we want to show -- how likely is that the performance on the Symbolic is a sample from the GSM8K distribution.  
Comparing two distributions, especially visually, is more complicated.

Reply:
yes agree, will revert to the original